debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/parser/LogMinerDmlEntryImpl.java

```diff
@@ -8,47 +8,57 @@
 import java.util.Arrays;
 import java.util.Objects;
 
-import io.debezium.connector.oracle.logminer.RowMapper;
+import io.debezium.connector.oracle.logminer.events.EventType;
+import io.debezium.connector.oracle.logminer.processor.infinispan.VisibleForMarshalling;
 
 /**
  * This class holds one parsed DML LogMiner record details
  *
  */
 public class LogMinerDmlEntryImpl implements LogMinerDmlEntry {
 
-    private final int operation;
+    private final EventType eventType;
     private final Object[] newValues;
     private final Object[] oldValues;
     private String objectOwner;
     private String objectName;
 
-    private LogMinerDmlEntryImpl(int operation, Object[] newValues, Object[] oldValues) {
-        this.operation = operation;
+    @VisibleForMarshalling
+    public LogMinerDmlEntryImpl(int eventType, Object[] newValues, Object[] oldValues, String owner, String name) {
+        this.eventType = EventType.from(eventType);
```

Comment:
Just so I understand, are you referring to having `LogMienrDmlEntryImpl#getEventType()` return the `EventType` but only have the class store the `int` value internally to avoid this extra lookup during loading from persistence storage?

Reply:
LogMinerDmlEntryImpl should accept `EventType eventType` param and the caller will call `EventType.from`

Reply:
ok, if this is really for serialization then it would add unnecessary complexity.